pom.xml

```diff
@@ -117,6 +121,33 @@
 			<artifactId>hapi-fhir-converter</artifactId>
 			<version>${hapi.version}</version>
 		</dependency>
+
+		<!-- Java 11 Dependencies -->
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+			<version>1.7.26</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-slf4j-impl</artifactId>
+			<version>2.8.2</version>
```

Comment:
There are newer versions of these libraries, but they didn't work. This was the latest version that worked.

Reply:
All of those at version 2.8.2 have 2.11.2.